MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -18,14 +18,22 @@ final class FeedViewModel {
 		Localized.Feed.title
 	}
 
+	var loadError: String {
+		Localized.Feed.loadError
+	}
+
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onError: Observer<Bool>?
```

Comment:
Just like in the MVP solution, the `var onError: Observer<Bool>?` could be defined as `var onErrorStateChange: Observer<String?>?`, so the ViewModel can decide when to show/hide the message and which message to show.

```suggestion
	var onErrorStateChange: Observer<String?>?
```

So, for example, in the `loadFeed()` method, the ViewModel can call  `onErrorStateChange?(.none)` to hide the error before loading the feed.

And on error, the View Model can call  `onErrorStateChange?(Localized.Feed.loadError)` to show the formatted error.

This way, the ViewController doesn't need to check a boolean value and then show or hide the message, and the View Model can decide to show different messages without having to alter the view controller. It's easier to manage presentation state transitions when all changes are centralized in the view model.